src/CodeXMLParse.cpp

```diff
@@ -27,6 +27,15 @@ struct ParseCodeXMLContext
 #define ANNOTATOR_PARAMS pugi::xml_node node, ParseCodeXMLContext *ctx, std::vector<RCodeAnnotation> *out
 #define ANNOTATOR [](ANNOTATOR_PARAMS) -> void
 
+static Varnode *getVarnodeForVariable(unsigned long long varref, ParseCodeXMLContext *ctx)
+{
+	for(auto it = ctx->func->beginLoc(); it != ctx->func->endLoc(); it++)
+	{
+		if((*it)->getCreateIndex() == varref)
+			return *it;
+	}
```

Comment:
Don't do this, better create a dictionary so that you can look up by varref without iterating through everything. In the worst case your current approach can take time quadratic to function size. Not a problem for most functions, but it might for bigger unrolled ones that you can often find in compression and crypto libraries.

Reply:
I will create a dictionary for this. Thanks.

Reply:
Done.